src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -636,6 +639,14 @@ def _get_protected_item_instance(item_type):
 
 
 def _check_map(item_type, item_type_map):
+    if item_type is None:
+        if item_type_map == workload_type_map:
+            raise RequiredArgumentMissingError("Workload type missing. Please enter a valid workload type.")
+        if item_type_map == protectable_item_type_map:
+            raise RequiredArgumentMissingError("""
+            Protectable item type missing. Please enter a valid protectable item type.
+            """)
+        raise RequiredArgumentMissingError("Item type missing. Enter a valid item type.")
```

Comment:
same as above

Reply:
It's just a placeholder. This case will never occur in today's code.